save_command_strategies/cmdline.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+PANE_PID="$1"
+CPID=$(pgrep -P $PANE_PID)
+
+exit_safely_if_empty_ppid() {
+	if [ -z "$PANE_PID" ]; then
+		exit 0
+	fi
+}
+
+full_command() {
+	[[ -z "$CPID" ]] && exit 0
+	cat /proc/${CPID}/cmdline | xargs -0 printf "%q "
```

Comment:
Since this strategy uses `procfs` I think a more fitting name for it would be `procfs` or maybe even `linux_procfs` to indicate it's linux only. I suggest renaming this file to `save_command_strategies/linux_procfs.sh`

Reply:
I think that's pretty reasonable; I'll go ahead and do that later today